pinot-clients/pinot-jdbc-client/src/main/java/org/apache/pinot/client/PinotDriver.java

```diff
@@ -68,6 +69,9 @@ public Connection connect(String url, Properties info)
       PinotClientTransport pinotClientTransport = factory.buildTransport();
       String controllerUrl = DriverUtils.getControllerFromURL(url);
       String tenant = info.getProperty(INFO_TENANT, DEFAULT_TENANT);
+      if(!headers.isEmpty()){
```

Comment:
nit: please add a space between `)` and `{`.

Reply:
Done